kong/pdk/log.lua

```diff
@@ -832,9 +832,9 @@ do
         },
         tries = (ctx.balancer_data or {}).tries,
         authenticated_entity = build_authenticated_entity(ctx),
-        route = ctx.route,
-        service = ctx.service,
-        consumer = ctx.authenticated_consumer,
+        route = utils.cycle_aware_deep_copy(ctx.route),
```

Comment:
Should we localize `utils.cycle_aware_deep_copy`?

Reply:
update